src/main/java/com/faforever/moderatorclient/api/dto/UserGroup.java

```diff
@@ -0,0 +1,28 @@
+package com.faforever.moderatorclient.api.dto;
+
+import com.faforever.commons.api.dto.AbstractEntity;
+import com.faforever.commons.api.dto.Player;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.github.jasminb.jsonapi.annotations.Relationship;
+import com.github.jasminb.jsonapi.annotations.Type;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Set;
+
+@Getter
+@Setter
+@Type(UserGroup.TYPE_NAME)
+public class UserGroup extends AbstractEntity {
+
+    public static final String TYPE_NAME = "userGroup";
+
+    private String technicalName;
+    private String nameKey;
+    private boolean public_;
```

Comment:
with _?

Reply:
`public` is a reserved keyword in Java, but not in MySQL. Do you have a better idea?